sailfishusermanager.cpp

```diff
@@ -287,6 +288,18 @@ uint SailfishUserManager::addUser(const QString &name)
     return uid;
 }
 
+int SailfishUserManager::removeUserFiles(const char *user)
+{
+     struct passwd *pwd = getpwnam(user);
+     if (pwd) {
+         QDir dir(USER_ENVIRONMENT_DIR.arg(pwd->pw_uid));
+         if (dir.removeRecursively())
```

Comment:
Should there be some check before going nucular - e.g. is clearing files for master user also ok?

Reply:
If dbus call was used this was already checked and denied but if ran directly or through userdel then the user is already lost.